### PR TITLE
Ready UI for Account page

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes

_Provide a detailed description of proposed changes._
- UI and dummy data for TxHistoryTable, Upload & Delete Tx History Modal, Tx Detail Drawer, và Download Tx History Button on Account page

### Proposed area of change

_Put an `x` in the boxes that apply._

- [x] `apps/bridge-dapp`
- [ ] `apps/hubble-stats`
- [ ] `apps/stats-dapp`
- [ ] `apps/tangle-dapp`
- [ ] `apps/faucet`
- [x] `libs/webb-ui-components`

### Reference issue to close (if applicable)

_Specify any issues that can be closed from these changes (e.g. Closes #233)._

- Related #1537 
- Related #1536 
- Related #1540 
- Related #1530 

### Screen Recording

_If possible provide a screen recording of proposed change._

https://github.com/webb-tools/webb-dapp/assets/69841784/57ca967e-e22d-4bc2-963b-600c4a2fdcd8

---

### Code Checklist

_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: apps/bridge-dapp/src/containers/TxDetailContainer/TxDetailContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+  const sourceTypedChainId = useMemo(() => {
+    // deposit
+    if (activity === 'deposit') {
+      return outputNotes.length > 0
+        ? +outputNotes[0].note.sourceChainId
+        : undefined;
+    }
+
+    // withdraw & transfer
```
Comment (drewstone) [inline apps/bridge-dapp/src/containers/TxDetailContainer/TxDetailContainer.tsx:63]: Are these comments accurate? They don't seem relevant to the code being commented. Also please use `// Capitals` when making comments.

File: apps/bridge-dapp/src/containers/TxDetailContainer/TxDetailContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    if (inputNotes.length === 0) return undefined;
+    return +inputNotes[0].note.targetChainId;
+  }, [activity, inputNotes, outputNotes]);
+
+  const destinationTypedChainId = useMemo(() => {
+    // deposit
```
Comment (drewstone) [inline apps/bridge-dapp/src/containers/TxDetailContainer/TxDetailContainer.tsx:69]: Is this comment really helpful? Confused what is being indicated here that isn't clear from the code below.

File: apps/bridge-dapp/src/containers/UploadTxHistoryModal/UploadTxHistoryModal.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                >
+                  Loaded File Successfully
+                </Typography>
+              </div>
+            </div>
+          )}
+        </div>
+
+        <ModalFooter>
+          {/* TODO: update onClick */}
```
Comment (drewstone) [inline apps/bridge-dapp/src/containers/UploadTxHistoryModal/UploadTxHistoryModal.tsx:89]: Can we create issues for TODOs and link in code?

File: apps/bridge-dapp/src/pages/Account/Transactions/TransactionDetail/index.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+            {/* TODO: Update to match with tx activity (Ex: Deposit Details) */}
+            Transaction Details
+          </Typography>
+          <DrawerCloseButton />
+        </div>
+
+        {/* Content */}
+
+        {/* DEPOSIT */}
+        {/* <TxDetailContainer
```
Comment (drewstone) [inline apps/bridge-dapp/src/pages/Account/Transactions/TransactionDetail/index.tsx:40]: Is this meant to be commented out and the withdraw not?

File: apps/bridge-dapp/src/pages/Account/Transactions/index.tsx [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+import HiddenValueEye from '../../../components/HiddenValueEye';
+import NoTx from '../NoTx';
+import TxTableContainer from '../../../containers/TxTableContainer';
+import { TxTableItemType } from '../../../containers/TxTableContainer/types';
+import downloadTxHistory, {
+  type DownloadTxType,
+} from '../../../utils/downloadTxHistory';
+
+import { randNumber, randEthereumAddress } from '@ngneat/falso';
+const fakeTxData: TxTableItemType[] = [
```
Comment (drewstone) [inline apps/bridge-dapp/src/pages/Account/Transactions/index.tsx:18]: Maybe we can extract fake data into a reusable file? It seems we're rewriting this in more than one place?

**Review by drewstone — CHANGES_REQUESTED** [review]
LG just some nits!

File: apps/bridge-dapp/src/containers/TxDetailContainer/TxDetailContainer.tsx (hunk 1 above)
Comment (vutuanlinh2k2) [reply apps/bridge-dapp/src/containers/TxDetailContainer/TxDetailContainer.tsx:63]: let me add more explanation there, @drewstone!

File: apps/bridge-dapp/src/containers/UploadTxHistoryModal/UploadTxHistoryModal.tsx (hunk 3 above)
Comment (vutuanlinh2k2) [reply apps/bridge-dapp/src/containers/UploadTxHistoryModal/UploadTxHistoryModal.tsx:89]: @drewstone, it's this issue #1540. This PR is only for preparing the needed UI for those mentioned tasks

File: apps/bridge-dapp/src/containers/TxDetailContainer/TxDetailContainer.tsx (hunk 2 above)
Comment (vutuanlinh2k2) [reply apps/bridge-dapp/src/containers/TxDetailContainer/TxDetailContainer.tsx:69]: same [here](https://github.com/webb-tools/webb-dapp/pull/1827#discussion_r1391937027)

File: apps/bridge-dapp/src/pages/Account/Transactions/TransactionDetail/index.tsx (hunk 4 above)
Comment (vutuanlinh2k2) [reply apps/bridge-dapp/src/pages/Account/Transactions/TransactionDetail/index.tsx:40]: It is **intended** here. 

The comment is for me to test in all 3 cases: DEPOSIT, TRANSFER, and WITHDRAW. I commented 2 out of 3 cases here. These will all be removed when integrating with the backend

File: apps/bridge-dapp/src/pages/Account/Transactions/index.tsx (hunk 5 above)
Comment (vutuanlinh2k2) [reply apps/bridge-dapp/src/pages/Account/Transactions/index.tsx:18]: These will all be removed when integrating with the backend. Keeping the fake data in each file will make it easier for us to remove them later. If creating a new file for reusable fake data, we have to delete it eventually.
